pkg/types/validation/installconfig_test.go

```diff
@@ -286,6 +286,34 @@ func validDualStackNetworkingConfig() *types.Networking {
 	}
 }
 
+func InvalidPrimaryV6DualStackNetworkingConfig() *types.Networking {
```

Comment:
This represent the scenario where keepalived was not happy because I forgot to change the order of `serviceNetwork` to have v6 primary address.